app/views/documents/edit.html.erb

```diff
@@ -55,6 +57,8 @@
         value: @document.summary,
         rows: 4,
         data: {
+          "module": "character-count",
```

Comment:
It looks like this kind of technique doesn’t extend if we wanted to add more modules to this element - would it be better to attach it in a similar way as contextual guidance?

Reply:
Yes, that is true, this method allows only one module per container, but generally I would avoid initialising multiple scripts on the same component as they may hijack each other's events. One script per component is consistent with how JavaScript is attached to all the components that are coming from `govuk-frontend`. If there are scripts acting at a document level, as it was the case of the contextual guidance, it makes sense for that script to use a `data-[component-name]` attribute selector or a `.js-[component-name]` class selector as we don't want to overlap with a component model and we want to be able to identify which script manipulates/affects the component. This is one of those things that I should document and explain at the project level.